Firmware/printer.cfg

```diff
@@ -58,23 +84,26 @@ uart_pin: PC11
 tx_pin: PC10
 uart_address: 2
 interpolate: True
-run_current: 0.5        # For V0.1 spec NEMA14 40Ncm
+run_current: 0.5
 hold_current: 0.25
 sense_resistor: 0.110
-stealthchop_threshold: 0
+stealthchop_threshold: 0                                            # Set to 9999 to turn on stealthchop
```

Comment:
@newphreak this comment isn't quite right. According to the Klipper docs, if specified that value should be `stealthchop_threshold: 999999` or `stealthchop_threshold: 0`.